instructions/Installation.md

```diff
@@ -42,16 +42,21 @@ At the end execute it:
 ```shell
 ./mac_czkawka_gui
 ```
+#### Compiling on Apple Silicon
+Prebuilt binaries are available only for x86_64, so if you use ARM e.g. Mac M1/M2, you must compile the app manually.  
 
-**Warning**  
-Prebuilt binaries are available only for x86_64, so if you use ARM machine like e.g. Mac M1, you need to compile manually app.  
+**Compilation is simple, and takes less than 10 minutes on an M1/M2 mac.**
```

Comment:
Out of curiosity, do you know why it's so long despite using these powerful M1/M2 CPU? Does it involve non-native code? Compiling czkawka with my 8 cores / 16 threads CPU takes 2 minutes on Ubuntu 22.04.

Reply:
Downloading libraries